src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ReleaseViewModels.cs

```diff
@@ -18,21 +19,21 @@ public class ReleaseViewModel
     {
         public Guid Id { get; set; }
 
-        public string Title { get; set; }
+        public string? Title { get; set; }
 
-        public string Slug { get; set;  }
+        public string? Slug { get; set; }
 
         public Guid PublicationId { get; set; }
 
-        public string PublicationTitle { get; set; }
+        public string? PublicationTitle { get; set; }
 
-        public string PublicationSlug { get; set; }
+        public string? PublicationSlug { get; set; }
 
-        public string ReleaseName { get; set; }
+        public string? ReleaseName { get; set; }
 
-        public string YearTitle { get; set; }
+        public string? YearTitle { get; set; }
```

Comment:
Why are all of these optional? These should be guaranteed to have values, so this is a strange change.

Reply:
I was getting warnings for these after adding `#nullable enable`. Adding stuff like `= string.Empty;` seemed unnecessary, but I didn't want to leave the warning, so I went for this.

Not sure what the best approach is in this case?

Reply:
Talked to Nick on Slack - I've switched to initialising them either as `string.Empty` or `null!` as appropriate.